benchmarks/functional_autograd_benchmark/functional_autograd_benchmark.py

```diff
@@ -36,6 +43,66 @@ def jacrev(model, inp, strict=None):
     else:
         return getattr(functional, task)
 
+def get_task_functorch(task: str) -> Callable:
+
+    def vjp(model, inp, v=None, strict=None):
+        assert v is not None
+        _, vjpfunc = ft.vjp(model, *inp)
+        return vjpfunc(v)
+
+    def jvp(model, inp, v=None, strict=None):
+        assert v is not None
+        return ft.jvp(model, inp, v)
+
+    def vhp(model, inp, v=None, strict=None):
+        assert v is not None
+        argnums = tuple(range(len(inp)))
+        _, vjpfunc = ft.vjp(ft.grad(model, argnums), *inp)
+        return vjpfunc(v)
+
+    def hvp(model, inp, v=None, strict=None):
+        assert v is not None
+        argnums = tuple(range(len(inp)))
+        return ft.jvp(ft.grad(model, argnums), inp, v)
+
+    def jacfwd(model, inp, v=None, strict=None):
+        argnums = tuple(range(len(inp)))
+        return ft.jacfwd(model, argnums)(*inp)
+
+    def jacrev(model, inp, v=None, strict=None):
+        argnums = tuple(range(len(inp)))
+        return ft.jacrev(model, argnums)(*inp)
+
+    def hessian(model, inp, v=None, strict=None):
+        return ft.hessian(model)(*inp)
+
+    def hessian_fwdrev(model, inp, v=None, strict=None):
+        return ft.jacfwd(ft.jacfwd(model))(*inp)
+
+    def hessian_revrev(model, inp, v=None, strict=None):
+        return ft.jacrev(ft.jacrev(model))(*inp)
+
```

Comment:
nit: we can rewrite the following but it might be harder to understand:
```
jacobian = jacrev
if task in ALLOWED_TASKS:
  return getattr(locals(), task)
raise RuntimeError("unsupported")
```